tpm2d/tpm2d.c

```diff
@@ -136,27 +159,15 @@ tpm2d_setup_keys(void)
 					ERROR("Failed to flush transient object handle of att primary key");
 				}
 				goto retry;
-			}
-			else {
+			} else {
 				dir_delete_folder(TPM2D_BASE_DIR, TPM2D_TOKEN_DIR);
 				FATAL("Failed to create attestation key with error code: %08x", ret);
 			}
-		}
-		else {
+		} else {
 			INFO("Created signing key for attestation in %s, not loading need to wait for provsg ...", token_dir);
 		}
-	}
-	else {
+	} else {
 		INFO("Signing key for attestation found in %s, loading ...", token_dir);
```

Comment:
don't we need a call here?

Reply:
nope that is what the patch is for that the load is not may during initialization but only when attestation request is made

Reply:
refactored tpm2d_setup_keys to make this clear.